manifests/database/postgresql.pp

```diff
@@ -10,20 +13,111 @@
     default => postgresql_password($::foreman::db_username, $::foreman::db_password),
   }
 
+  $rep_password = $::foreman::db_replication_password ? {
+    'UNSET' => false,
```

Comment:
I strongly dislike the UNSET we already have but dropping them has been too much of an effort until now. Let's at least try to avoid adding more.